sql/core/src/test/scala/org/apache/spark/sql/execution/columnar/InMemoryColumnarBenchmark.scala

```diff
@@ -59,8 +60,9 @@ object InMemoryColumnarBenchmark extends SqlBasedBenchmark {
   }
 
   override def runBenchmarkSuite(mainArgs: Array[String]): Unit = {
+    val rowsNum = if (mainArgs.length > 0) mainArgs(0).toLong else 1000000
     runBenchmark("Int In-memory") {
```

Comment:
Should we put `rowsNum` to the title?

Reply:
Yes, I added it to the PR title.

Reply:
Ah, do you mean `benchmark` title? Got it.

Reply:
Yea, Thanks.